apps/condo/domains/acquiring/schema/AcquiringIntegration.js

```diff
@@ -86,6 +86,12 @@ const AcquiringIntegration = new GQLListSchema('AcquiringIntegration', {
             options: CONTEXT_STATUSES,
             defaultValue: CONTEXT_IN_PROGRESS_STATUS,
         },
+        vatPercentOptions: {
```

Comment:
Should be string or typed JSON? If there any validation for field format?

Reply:
Added a resolving hook + tests